client/src/Components/BootcampApplications/BootcampApplications.jsx

```diff
@@ -1,12 +1,58 @@
-import React, { Component} from 'react';
-
-class BootcampApplications extends Component {
-
-  render() { 
-    return ( 
-      <h1>This is the BootcampAPPLICATIONS component</h1>
-    );
-  }
-}
- 
-export default BootcampApplications;
\ No newline at end of file
+import React, { useState, Fragment} from 'react';
+import axios from "axios";
+
+const BootcampApplications = () => {
+  const [state, setState] = useState({
+    date_applied: new Date(),
+    rejected: false,
+    accepted: false,
+    applicationSubmitted: false
+  });
+
+  const handleChange = e => {
+    const { name, value } = e.target;
+    setState({ ...state, [name]: Number(value) });
+  };
+
+  const handleSubmit = async e => {
+    e.preventDefault();
+    const endpoint = "http://localhost:3000/job-applications/add-application";
+
+    const payload = {
+      date_applied: state.date_applied,
+      rejected: state.rejected,
+      accepted: state.accepted,
+      users_id: 1, /* we need to pull this from session or something similar */
```

Comment:
the pending Login and signup PR will fix this as we can pull and set a user state from the redux store